mono/metadata/locales.c

```diff
@@ -294,20 +300,20 @@ construct_culture (MonoCultureInfo *this_obj, const CultureInfoEntry *ci)
 }
 
 static MonoBoolean
-construct_region (MonoRegionInfo *this_obj, const RegionInfoEntry *ri)
+construct_region (MonoRegionInfoHandle* this_obj, const RegionInfoEntry *ri)
```

Comment:
I don't really like the idea of `MonoHandleStorage`, that just add an indirection/type for no added value.